src/Core/Enum.php

```diff
@@ -69,20 +71,20 @@ public function has($value)
     *
     * @return bool True if the value is defined
     */
-    public function is($value)
+    public function is(string $value): bool
     {
         return $this->_value === $value;
     }
 
 	/**
 	 * Create a new class for the enum in question
 	 *
-	 * @return mixed
+	 * @return array
 	 * @throws \ReflectionException
 	 */
     public function toArray()
     {
-        $class = get_called_class();
+        $class = static::class;
```

Comment:
My understanding is both `get_called_class()` and `static::class` yield the same results?

Reply:
Correct, but `static::class` is PHP 5.5+ and `get_called_class()` has been proposed to be deprecated in PHP 8.1 (not confirmed yet)

Reply:
Got it, thanks